test/FsAutoComplete.Tests.Lsp/EmptyFileTests.fs

```diff
@@ -33,15 +33,15 @@ let tests state =
         [
             testCaseAsync
                 "no parsing/checking errors"
-                (async {
+                <| async {
```

Comment:
Not a necessary change but showing that we're still missing some extremely complex syntax pattern match to be able to see the runnable part on the side. (See nothing next to `async ({ .. })` vs `async <|` having an icon)

![image](https://github.com/fsharp/FsAutoComplete/assets/1490044/89e5ed95-8f6b-4673-9dd3-e83627ced84a)